addons/binding/org.openhab.binding.yeelight/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+    <config-description uri="thing-type:device:base">
+        <parameter name="deviceId" type="text" required="true">
+		    <label>Device ID</label>
+		    <description>Id of the Yeelight device to connect with.</description>
+		</parameter>
+		        <parameter name="duration" type="integer" required="true" min="50" max="5000">
```

Comment:
The indentation here looks wrong. I assume this has something to do with mixing tabs and spaces. If you use Eclipse as described in the documentation for developing bindings for openHAB, you should be able to simply apply auto formatting in order to get this into the standard style for openHAB.

Reply:
Fixed. Also extends max duration to 10s.